src/rdb_protocol/op.hpp

```diff
@@ -18,22 +18,24 @@ namespace ql {
 
 class func_term_t;
 
-// Note that worst_determinism depends on the order of these; if you want to add
-// a new value, make sure worst_determinism is properly defined for it.
-enum class deterministic_t {
```

Comment:
Is it now possible for `DET_NONDET | DET_SINGLE_SERVER` to be a value of deterministic_t?  Is it possible for `DET_NONDET` to be a value?